### PR TITLE
Update README.md for native instructions

### PR DESCRIPTION
**Pull request by sonya**
A first pass at react native instructions so people can get started

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,5 +75,38 @@ re-run `docker-compose run app bundle install` or the `rake` commands
...
+You have to be on OS X.
+
+You should also have Node.js and `npm` installed.
+[Here is a good set of instructions for OS X](http://shapeshed.com/setting-up-nodejs-and-npm-on-mac-osx/)
+
+You should also install the [following things](https://facebook.github.io/react-native/docs/getting-started.html):
+
+    `brew install watchman`
+    `brew install flow`
+    `npm install -g react-native cli`
```
Comment (mxie) [inline README.md]: If these lines are 4 spaces in, there's no need for the backticks. They'll automatically format correctly.

Comment (sonya) [inline README.md]: do you want me to fix that? i'm not sure if this branch is being pulled or not

File: README.md [hunk 2]
```diff
@@ -75,5 +75,38 @@ re-run `docker-compose run app bundle install` or the `rake` commands
...
+
+Finally you should have [Xcode](https://itunes.apple.com/us/app/xcode/id497799835?mt=12) installed, with the latest iOS simulators set up via Preferences.
+
+### iOS
+
+`cd` to the `EnerSave` directory and run `npm install` (you should only need to do this once)
+
+Open Xcode.
+
+Open the .xcodeproj file which you should find in the `EnerSave/ios` directory.
```
Comment (mxie) [inline README.md]: Format `.xcodeproj` maybe?

File: README.md (hunk 1 above)
Comment (mxie) [inline README.md]: Being pulled in what sense?

Also, are we actually trying to install the `cli` package or `react-native-cli`? I think the difference _might_ be causing all the build warnings/errors I'm seeing in Xcode.

Comment (sonya) [inline README.md]: sorry, merged? is that the right term? accepted? i figured it's a "pull request" so someone would, you know, pull it

and yes that is a typo, it should be `react-native-cli`. sorry about the build warnings.

**Comment by sonya** [comment]
made changes as requested.

**Comment by mxie** [comment]
Looks great!

**Pull request merged**